package.json

```diff
@@ -13,7 +13,8 @@
     "lint:fix": "eslint src --fix && prettier --write src",
     "format": "prettier --write src",
     "test": "vitest",
-    "coverage": "vitest run --coverage",
+    "coverage": "vitest run --coverage.enabled --coverage.all",
+    "coverage:covered": "vitest run --coverage",
```

Comment:
what this command mean?

Reply:
"coverage" - show coverage for all typescript files ("all" option)
"coverage:covered" - only show coverage for files with tests (useful for development)